code/popup.html

```diff
@@ -9,6 +9,7 @@
   <link rel="stylesheet" type="text/css" href="css/font.css">
   <link rel="stylesheet" type="text/css" href="css/fontawesome-all.min.css">
   <link rel="shortcut icon" type="image/png" href="/icon/icon.png"/>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
We can link only to local resources as this extension is intended to work offline. We are using `fontawesome-all.min.css`, so, you may consider using this as we must consider the size of our extension as well.